Dockerfile

```diff
@@ -1,11 +1,32 @@
-FROM node:16.13.1-alpine3.15
+# syntax=docker/dockerfile:1
+FROM node:lts-alpine3.17 as builder
 
-WORKDIR /usr/src/app
+WORKDIR /app
 
 COPY package*.json ./
+COPY prisma ./prisma
+
 RUN npm install
 
 COPY . .
-RUN npx prisma generate && npx prisma migrate dev --name init && npm run build
 
-CMD ["npm", "run", "serve"]
+RUN npm run build && npx prisma generate
+
+FROM node:lts-alpine3.17
+
+WORKDIR /app
+
+RUN addgroup -S twiggy && adduser -S twiggy -G twiggy
+
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/assets ./assets
+COPY --from=builder /app/package*.json ./
+COPY --from=builder /app/build ./build
+COPY --from=builder /app/prisma ./prisma
+COPY --from=builder /app/prisma/database.empty ./prisma/db/main.db
+
+RUN chown -R twiggy:twiggy /app/prisma/db /app/assets/NFD/images
```

Comment:
I think this line could be removed by adding `COPY --chown=twiggy:twiggy` to the previous appropiate `COPY` commands

Reply:
I only want to have specific paths accessible for the twiggy user.